tests/unit/admin/test-class-wc-payments-admin.php

```diff
@@ -128,6 +117,35 @@ public function test_it_does_not_render_payments_badge_if_stripe_is_connected()
 		$this->assertArrayNotHasKey( 'wc-admin&path=/payments/connect', $item_names_by_urls );
 	}
 
+	public function test_it_renders_payments_badge_if_activation_date_is_older_than_7_days() {
+		global $menu;
+		$this->mock_current_user_is_admin();
+
+		// Make sure we render the menu with submenu items.
+		$this->mock_account->method( 'try_is_stripe_connected' )->willReturn( false );
+		update_option( 'wcpay_activation_timestamp', time() - ( 86400 * 7 ) );
```

Comment:
Super nitpicky, but just an idea for clarity. 😄 

```suggestion
		update_option( 'wcpay_activation_timestamp', time() - WEEK_IN_SECONDS );
```

Reply:
Not a nitpick, I totally missed updating it on here. I'll make an update 👍